cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConfiguration.java

```diff
@@ -160,5 +166,48 @@ public boolean anyReadOnly(KafkaVersion kafkaVersion) {
         return result;
     }
 
+    /**
+     * Returns true if the supplied value is default to kafka in kafkaVersion version
+     * @param property name of the tested property
+     * @param value desired value
+     * @param kafkaVersion version of Kafka
+     * @return true if the desired value is default
+     */
+    @SuppressWarnings("checkstyle:Regexp")
+    public static boolean isValueOfPropertyDefault(String property, String value, KafkaVersion kafkaVersion) {
+        value = value == null ? "null" : value;
+        Map<String, ConfigModel> c = readConfigModel(kafkaVersion);
+        Optional<Map.Entry<String, ConfigModel>> optProp = c.entrySet().stream().filter(entry -> entry.getKey().equals(property)).findFirst();
+        if (optProp.isPresent()) {
```

Comment:
`c` is a `Map<configName → model >`, right? So why the linear search the matching model, when you could just do `c.get(property)`?